stk/molecular/molecules/macrocycles_rotaxanes.py

```diff
@@ -0,0 +1,133 @@
+"""
+Defines classes that describe macrocycles and rotaxanes.
+
+There is a family of classes dealing with macrocycles, a cyclic
+oligomer topology used to construct macrocycles, and classes related
+to rotaxanes.
+
+A base class :class:`MacrocycleBase` contains the methods used by all
+macrocycles, no matter whether loaded or constructed using ``stk``.
+A daugter :class:`MacrocycleStructUnit` also inherits
```

Comment:
daugter, also I would prefer the term "child class" to be consistent with the rest of the documentation.